extensions/amp-twitter/0.1/amp-twitter.js

```diff
@@ -71,8 +71,15 @@ class AmpTwitter extends AMP.BaseElement {
       if (this.getFallback()) {
         this.togglePlaceholder(false);
         this.toggleFallback(true);
+      } else {
```

Comment:
the issue only says placeholder, but I think we should also do it for fallback. (+ add an example with fallback to the example file).

Reply:
Done. 
1. Added the code to resize if fallback is provided
2. Added an example as well.